### PR TITLE
Add search function to send message to every contact(even if at last) and add ability to choose FireFox or Chrome driver

### PR DESCRIPTION
**Pull request by IamMU**
# List Of Changes:
 - Fix issue #6 (Working and tested a lot)
 - Fix issue #5 (Not fully tested)
# Additional Libraries:
 - None

### CODE REVIEW

File: whatsappbomber.py [hunk 1]
```diff
@@ -135,50 +137,138 @@ def start_bot(names: list, messages: list, driver: any) -> None:
...
+
+            # Loop through messages list and send them one by one
+            log("info", "Sending messages", hierarchy_level=2)
+            for message in messages:
+                log(f"info", f"Writing: {message}", hierarchy_level=3)
+                text_box.send_keys(message)
+
+                # Wait for brief moment before sending to ensure full reliability
+                log(f"info", f"WAITING", hierarchy_level=4)
+                time.sleep(0.4)
```
Comment (mrpandya) [inline whatsappbomber.py:167]: why do we need time.sleep()?

File: whatsappbomber.py [hunk 2]
```diff
@@ -135,50 +137,138 @@ def start_bot(names: list, messages: list, driver: any) -> None:
...
+
+                log(f"info", f"Sending the message...", hierarchy_level=4)
+                # Get send button element
+                # send_button = wait.until(EC.presence_of_element_located(("xpath", "/html/body/div[1]/div/div/div[4]/div/footer/div[1]/div/span[2]/div/div[2]/div[2]/button")))
```
Comment (mrpandya) [inline whatsappbomber.py:171]: remove unnecessary comments

**Review by mrpandya — CHANGES_REQUESTED** [review]
please remove unnecessary comments and also we do not need the log file to be in our repo.
along with this do mention in the readme that the user has to add the geckodriver in the folder or else upload the geckodriver and create a new PR. I'd suggest you upload the geckodriver just like you uploaded the chromium driver. Also I didn't notice that there was already a chromedriver in the repo so if possible remove the older version and make another PR regarding the same, I will add the hacktoberfest-accepted label for the same.

File: whatsappbomber.py (hunk 2 above)
Comment (IamMU) [reply whatsappbomber.py]: Done

**Comment by IamMU** [comment]
I'll upload the driver to the same pr

**Comment by IamMU** [comment]
See if all files are there. I put the geckodriver and a new version of chromedriver in the commit

**Review by mrpandya — APPROVED** [review]

**Pull request merged**